src/Entity/Package.php

```diff
@@ -3,13 +3,20 @@
 namespace AppBundle\Entity;
 
 use Gedmo\Timestampable\Traits\Timestampable;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 class Package
 {
     use Timestampable;
 
     protected $id;
+    /**
+     * @Groups({"task", "delivery"})
+     */
     protected $name;
+    /**
+     * @Groups({"task", "delivery"})
```

Comment:
At the moment it's not useful to serialize the `volumeUnits` property. It's something that is yet pretty unused (except very recently in #3118), it was introduced to be used in a near future for [route optimization using VROOM](https://github.com/VROOM-Project/vroom/blob/master/docs/API.md#shipments). 

But in any case even if it was used for route optimization, API consumers wouldn't care that much about the volume units of each packages, it's more for internal purpose.

Reply:
Ok, I just removed `volumeUnits` from the response.